### ISSUE TITLE
Lizenz: Momentan ein no-go

### ISSUE THREAD

**Issue by trollvottel** [ioBroker/ioBroker.knx#209]
Erstmal müsste der Adapter wirklich stabil laufen. Dazu bräuchte es mal mindestens eine trial-periode für v2.0!

Beim Import meiner .knxproj Datei kommt am Ende irgendwas von >3000 states importiert. Da würde für mein Einfamilienhaus also nichtmal die "größte" Lizenz ausreichen, na alles klar!

**Comment by chefkoch009** [comment]
Sind es denn mehr als 3000 dazenpunkte? Könnte ich die Projektdatei zur Analyse bekommen?

**Comment by trollvottel** [comment]
Klar, kann ich Dir gerne schicken (private email?)

**Comment by trollvottel** [comment]
Ich weiß nicht, was Du mit "Datenpunkten" meinst, ich kenne nur DPT als Datentyp und GA als Gruppenadressen im KNX.

Ich würde auch garnicht alle Gruppenadressen benötigen, aber es wird halt erstmal alles importiert, oder?

**Comment by JohnWayne007** [comment]
Gibt es keine Community Lizenz mehr? Bei IoBroker.net sehe ich nur die Kommerziellen Lizenzen.

**Comment by Anzic23** [comment]
Ich Stimme mit @trollvottel und @JohnWayne007 kann, lohnt sich eine Lizenz für die Gemeinschaft mindestens 300-500 Datenpunkten? So zumindest wird es möglich sein, zu testen die zweite Version des Adapters, nicht kaufen die gleiche 'Katze im Sack' (Russisches Sprichwort).
I'm not sure that the translation from Russian to German is quite accurate, so I'll duplicate it in English:
I agree with @trollvottel and @JohnWayne007 maybe we should make a license for the community for at least 300-500 datapoints? So at least it will be possible to test the second version of the adapter, but do not buy a 'cat in a poke' (Russian proverb).

**Comment by chefkoch009** [comment]
500 DP aka Groupadresses (GA) are completly free. You don't need a license for that. If you don't have installed a license, then you can use up to 500 DP.

**Comment by chefkoch009** [comment]
@trollvottel : schreib mir im iobroker forum bitte eine PN.

**Comment by Missenberger79** [comment]
kann man bei einem wechsel des Hosts diese dann wechseln oder muss dann erneut bezahlt werden?
gibt es ein upgrade von 1000 auf 2000 wenn diese dann nötig wird?

**Comment by chefkoch009** [comment]
>>kann man bei einem wechsel des Hosts diese dann wechseln oder muss dann erneut bezahlt werden?
Natürlich kann die Lizenz mitgenommen werden, muss aber für den neuen Host neu erstellt werden.

>> gibt es ein upgrade von 1000 auf 2000 wenn diese dann nötig wird?
Zur Zeit nicht, dieser Punkt befindet sich noch in Klärung.

>>was passiert wenn ich eine 1000er habe und nach einem GA-Update 1012 hätte, stoppt der Adapter?
Ja, es gilt bis 1000 .

**Comment by Missenberger79** [comment]
> > > kann man bei einem wechsel des Hosts diese dann wechseln oder muss dann erneut bezahlt werden?
> > > Natürlich kann die Lizenz mitgenommen werden, muss aber für den neuen Host neu erstellt werden.
> 
> > > gibt es ein upgrade von 1000 auf 2000 wenn diese dann nötig wird?
> > > Zur Zeit nicht, dieser Punkt befindet sich noch in Klärung.
> 
> > > was passiert wenn ich eine 1000er habe und nach einem GA-Update 1012 hätte, stoppt der Adapter?
> > > Ja, es gilt bis 1000 .

@chefkoch009:
Danke für die Antworten. Grundsätzlich hätte ich kein Problem für den Adapter zu bezahlen. Ich kann einschätzen was da Arbeit dahinter steckt. Jedoch finde ich, das man hier die Stufen „großzügiger“ Auslegen könnte und dafür nur zwei Stufen.  Hab mit meinem Projekt, ein in meinen Augen einfaches EFH, schon das Problem in die zweite Stufe zu kommen. Und da muss ich mir schon wirklich Gedanken zu machen wenn die dann fehlende Lizenz beim Überschreiten der GA den Adapter blockiert und ein Upgrade ja noch in Sterne steht es mir wert ist. Da für die nächste Stufe wieder den vollen Preis zu zahlen. 

Wäre es nicht alternativ möglich z.b das Lizenzmodel auf bis 2 Linien oder 128 Geräte zu machen. 

Oder z.b 10€ für 100 weitere GA etc.

**Comment by chefkoch009** [comment]
>>Vielleicht gehst du da nochmal in dich?
Da war ich schon oft gewesen. Deine Idee hatte ich auch schon, aber egal wie man es dreht und wendet, es ist anders nicht mehr zu stämmen. Ich möchte den Adapter unbedingt weiter entwickeln und hatte naiver Weise auf die Ehrlichkeit einiger Profis gehofft. Naja....der Rest ist Geschichte. Es ging und geht mir nie um die Zeit...das ist jenseits von Gut und Böse....aber das ist bei den anderen Entwicklern auch nicht anders.

**Comment by trollvottel** [comment]
Ich finde die jährliche subscription preislich ok. Es ist zwar bitter, dass man mit einem wirklich smarten EFH bereits die 2000er Lizenz braucht, aber z.B. bei Gira X1 ist die Grenze auch bei 1000.

Was mich eher fuchst ist die Tatsache, dass man sein Projekt bzw. ein Upgrade des Adapters nicht einfach für ein paar Tage mal ausprobieren kann, sondern direkt ne Lizenz braucht. v1 war ziemlich buggy und habe mich oft darüber geärgert, wenn ich dafür jetzt noch bezahlen soll, wechsle ich lieber gleich die ganze Basis, zu z.B. Home Assistant.

Ich habe gestern den Tipp erhalten, vor dem Import mein ETs Projekt zu klonen und im Klon großflächig GAs rauszulöschen, die ich eh nicht brauche. Das kann man ein mal machen, aber spätestens beim 2. mal nervt das tierisch, wenn man unter den 500 bleiben muss weil man kostenlos bleiben will.

**Comment by Missenberger79** [comment]
Beim X1 habe ich aber die Möglichkeit die GA die benötigt werden für das Projekt im Gira Tool in die Projektierung zu übernehmen. 
hier muss ich dann vorher mit einem Klon alles rauslösenden was ich evtl. nicht brauch. 
Um dann den Klon wieder zu übernehmen. 

Diese Vorgehensweise ist ja einmalig noch ok. 
aber wenn ich jetzt ein paar Dinge ändere nervt es mich schon beim Gedanken. 
vielleicht wäre es hier möglich über das neue GA Tool eine Manuelles Anlegen zu machen oder das würde mich freuen wenn einfach das 2000er als Einsteigerlizenzmodel startet. 500er frei. Das 1000er ist das einzige was mir hier, weil ich mit ein paar Lichter Jalousie und Heizung schon drüber komm.

**Comment by chefkoch009** [comment]
@Missenberger79 Der Gedanke klingt interessant. Das nehme ich mal als Feature Wunsch auf.

**Comment by barni** [comment]
Hallo zusammen,

Ich kann die Diskussion nur bedingt nachvollziehen. Nachvollziehen kann ich so Faktoren wie, wienlange wird vermutlich eine Version unterstützt, wie sieht es beim Umzug der Hardware, etc. aus.
Aber ob 500 GAs zu 14 Euro im Monat oder 2000 GAs zu 26 Euro im Monat die richtigen Grenzen und Preise sind ist total subjektiv und was etwas Wert ist wird jeder anders bewerten. Demnach könnte man da endlos diskutieren.
Ich für mich sage das selbst 2000 GAs günstiger sind als eine Packung Klopapier im Moment und Entwicklung/Arbeit was Wert ist.

Im übrigen habe ich gestern perfekten Support bezogen auf diesen Adapter erlebt.
Grüße

**Comment by trollvottel** [comment]
Ja, das fände ich gut. Denn **wirklich im ioBroker nutzen** tue ich sehr viel weniger, nichtmal 100! Dann wären auch andere Limits denkbar.

**Comment by Missenberger79** [comment]
Bzgl des Klon und gelöschte GA in der ETS wäre dann ein Einstellmöglichkeiten ganz gut um die Fehlermeldung über nicht vorhandene GA zu unterdrücken.

**Comment by trollvottel** [comment]
Klon und GAs löschen ist ein ganz mieser Workflow. Bitte nicht darauf aufbauen.

**Comment by iobroker99** [comment]
@chefkoch99, wie finde ich raus wieviele Datenpunkte ich habe, d.h. welche Lizenz ich brauche? Ist 1 DP = 1 GA so wie man oben aus einer Antwort von Dir rauslesen könnte?

**Comment by trollvottel** [comment]
Ja, GA = Datenpunkt im Sinne des Adapters.

**Comment by chefkoch009** [comment]
@iobroker99 In der ETS unter Reports den Punkt Projektstatistik auswählen und dann Seite 2 Abschnitt Gruppenadressen.

VG
chefkoch009

**Comment by iobroker99** [comment]
@chefkoch99, danke. Preismodell finde ich eigentlich ok. Ich habe ziemlich durchautomatisierte Häuser - das Ferienhaus hat knapp 500 GAs, das Wohnhaus 700. Um die 100 Euro pro Jahr für so einen essentiellen Adapter passt für mich, wenn er denn gut läuft

**Comment by chefkoch009** [comment]
@iobroker99 : 
>>wenn er denn gut läuft
Da arbeite ich dran.

**Comment by iobroker99** [comment]
@chefkoch009 

> > > wenn er denn gut läuft
> > > Da arbeite ich dran.
lizenz gekauft, installiert, ausprobiert. läuft gut. bislang nichts gefunden was sich komisch verhält. und der GA Editor ist sehr fein. Danke :-)

**Comment by ChrisDuck87** [comment]
Was habt ihr alles für extreme KNX-Installationen. Hab bei mir auch eine KNX Installation aber ich komme grad mal 311 Gruppenadressen....

**Comment by trollvottel** [comment]
Äh wohl eher andersrum: Dann hast Du ne mickrige KNX Installation mit kaum integrierten Gewerken.

Hier Licht inkl. RGBW und Präsenzmeldern überall, jede Menge schaltbare Steckdosen 8 mit Strommessung, Beschattung für 18 Behänge  viele davon Jalousien mit Lamellenverstellung und Sonnenstandsnachführung, Heizung Einzelraumregelung und Wärmepumpenansteuerung, Zentrale Lüftung, Energiemanagement mit PV/Speicher/Wallbox/Tesla/SGReady/Waschtrockner/Geschirrspüler, diverses IoT Zeugs wie Automower und Bewässerung, Alarme, Wetterstation, Diagnose inkl. Bus, separate Aussenlinie für Garage und diverse Logiken usw. Da kommt bei nem wirklich (!) smarten EFH schon was zusammen!

**Comment by chefkoch009** [comment]
Nunja .... über die grösse lässt sich bekanntlich streiten ;-) .... Ich habe super interessante Installationen mit weniger bis massiv vielen DP gesehen. Einige davon waren echt stilvoll und wohl überlegt, andere nur hingewurstelt..... ich denke über die Anzahl der DP kann man keine Aussage über die Anlage treffen. Andererseits ist es auch so, das gerade im EFH Sektor die Anlagen "historisch" Wachsen.

**Comment by ChrisDuck87** [comment]
ja sehe es wie @chefkoch009: wenn ich mir die Liste von Trollvottel durchlese habe ich eigentlich (fast) ebenfalls alles aber halt nur einen Bruchteil (dank Iobroker ;) )  über KNX gelöst. Will hier jetzt aber keinen Wettbewerb/Diskussion auslösen. Mich hats nur gewundert. Mir reicht da meine mickrige Anlage ;)

PS: 2.0 Funktioniert bei mir einwandfrei! Danke!

**Comment by iobroker99** [comment]
Naja. Sowohl Ferienhaus als auch Wohnhaus bei mir haben jeweils: Einzelraumsteuerung, diverse Lampen mit Dimmern je Zimmer, diverse RGBs, Wärmepumpe mit KNX Gateway, Lüftung, Jalousien mit Lamellen, PV mit Akku, Wallcharger, Alarm, schaltbare Steckdosen mit Strommessung sowieso. Im Wohnhaus noch Kino, Partyraum, ELW mit eigener Linie. Es ist nicht smarter, nur weil man mehr GAs verballert ;-) Datensparsamkeit sag ich da nur. Und ob man Logiken in KNX abbildet muss jeder selbst wissen, ich steh nicht drauf. Wenn ich die GAs optimieren würde, bräuchte ich wahrscheinlich inm beiden Häusern keine Lizenz. 500 GA ist schon eine echte Menge wenn man ordentlich damit umgeht.

**Comment by trollvottel** [comment]
Der Punkt ist: Abgesehen von dem IOT zeugs funktioniert das bei mir alles auch ohne Server bzw. ioBroker. Ja für Logiken in KNX braucht es deutlich mehr Adressen.

**Comment by chefkoch009** [comment]
@trollvottel Das sehe ich genau so. Die Basis muss unter fasst allen Umständen funktionieren. Wenn ich ohne z.B.  einen Sprachassistenten mein Licht nicht mehr anbekomme oder meine Heizung gar mehr nicht funktioniert, dann ist das gar nicht gut.

**Comment by iobroker99** [comment]
>Meiner Ansicht nach ist ein Smarthome, das bei Auszug schlagartig dumm wird, kein echtes Smarthome. Das Ganze >muss Übergebbar sein.

die Basis funktioniert bei mir ebenfalls ohne ioB. Das Haus wird auch nicht dumm ohne ioB, aber es verliert energetische Optimierungen. 
In der Realität habe ich bislang eher wenige KNX zertifizierte Elektriker kennengelernt, die mehr als den KNX 08/15 Standard beherrschen. Wieviele davon komplexere KNX Logiken übernehmen können und dann auch noch Bock darauf haben, das für eine am Ende doch relativ kleine Anlage (nämlich Haus, nicht großer Industriebau) zu machen - naja.

**Comment by trollvottel** [comment]
Ich habe fairnesshalber mal den Titel des issues abgeändert.

**Comment by chefkoch009** [comment]
Genaugenommen ist das ein issue und bugtracking System und keine Diskussionsplattform. Dieses Thema ist eine Diskussion und keine Fehlermeldung. Desweiteren wurde und wird diese Diskussion im Forum bereits geführt. Deshalb schließe ich diesen nicht-Issue.
